contrib/generate-ds20.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name,missing-docstring
+#
+# Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# pylint: disable=consider-using-f-string
+
+import sys
+import argparse
+import configparser
+import base64
+from typing import List
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-b",
+        "--bufsz",
```

Comment:
How about making this argument optional and default to a rounded number based on quirk data?
Then you can print at the end that a buffer of X bytes was used, if you need it to be bigger, use the `--buffsz` argument